### PR TITLE
Re enable tests that were unavailable during maintenance

### PR DESCRIPTION
**Pull request by paulkaplan**
/cc @kchadha @rschamp

### CODE REVIEW

**Review by kchadha — CHANGES_REQUESTED** [review]
This also enables a test that was being skipped from before maintenance -- the util_task_queue test. That one was being skipped because it's a flaky test.

File: test/unit/util_task-queue.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-const test = require('tap').skip;
+const test = require('tap').test;
```
Comment (kchadha) [inline test/unit/util_task-queue.js]:
```suggestion
const test = require('tap').skip;
```

Comment (paulkaplan) [reply test/unit/util_task-queue.js]: Gotchya thanks

**Review by kchadha — APPROVED** [review]
LGTM

**Pull request merged**